src/elements/content-sidebar/BoxAISidebar.tsx

```diff
@@ -13,69 +13,74 @@ import { DOCUMENT_SUGGESTED_QUESTIONS, SPREADSHEET_FILE_EXTENSIONS } from '../co
 import messages from '../common/content-answers/messages';
 
 export interface BoxAISidebarContextValues {
-    cache: { encodedSession?: string | null, questions?: QuestionType[] },
-    contentName: string,
-    elementId: string,
-    recordAction: (params: RecordActionType) => void,
-    setCacheValue: (key: 'encodedSession' | 'questions', value: string | null | QuestionType[]) => void,
-    userInfo: { name: string, avatarUrl: string },
-};
+    cache: { encodedSession?: string | null; questions?: QuestionType[] };
+    contentName: string;
+    elementId: string;
+    isStopResponseEnabled: boolean;
```

Comment:
wondering shouldn't it be optional if we then set the default value to false?

Reply:
and is it possible to test it in storybook that stop button appears once question is sent?